physics/unified_ugwp_post.F90

```diff
@@ -0,0 +1,83 @@
+!>  \file unified_ugwp_post.F90
+!! This file contains
+module unified_ugwp_post
+
+contains
+
+!>\defgroup unified_ugwp_post unified_UGWP Scheme Post
+!! @{
+!> \section arg_table_unified_ugwp_post_init Argument Table
+!!
+    subroutine unified_ugwp_post_init ()
+    end subroutine unified_ugwp_post_init
+
+!>@brief The subroutine initializes the unified UGWP
+#if 0
```

Comment:
We don't need lines 15 (`#if 0`) and 19 (`#endif`) anymore.

Reply:
Deleted lines 15 and 19.